airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/AbstractJdbcCompatibleSourceOperations.kt

```diff
@@ -332,8 +332,10 @@ abstract class AbstractJdbcCompatibleSourceOperations<Datatype> :
         }
     }
 
+    // Note: I'm guessing this is an open function, since it doesn't do anything here, and doesn't
+    // override anything either
```

Comment:
I can answer that: they're all open.

Reply:
I think they should be abstract. I really don't like optional methods. Battle for another day though